web-sdk/v3.md

```diff
@@ -628,3 +628,156 @@ Here are some examples of what is achievable using the customization:
     { buttonText: 'Theme 2', slug: 'OJbxZzW', title: 'V3 Fidel Web SDK' },
     { buttonText: 'Theme 3', slug: 'eYBGrrQ', title: 'V3 Fidel Web SDK' },
 ]} />
+
+## Card entity
+
+This object is returned through the `onCardEnrolledCallback` when a card is successfully enrolled.
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td><code>id</code></td>
+    <td>string</td>
+    <td>ID of the enrolled card</td>
```

Comment:
For this description and the next 2 fields, would be clearer to say:
"The ID..."
"The Account ID..."
"The ID of the program..."